enterprise_gateway/services/processproxies/yarn.py

```diff
@@ -33,9 +33,20 @@ class YarnClusterProcessProxy(RemoteProcessProxy):
     def __init__(self, kernel_manager, proxy_config):
         super(YarnClusterProcessProxy, self).__init__(kernel_manager, proxy_config)
         self.application_id = None
-        self.yarn_endpoint = proxy_config.get('yarn_endpoint', kernel_manager.parent.parent.yarn_endpoint)
+        self.yarn_endpoint \
+            = proxy_config.get('yarn_endpoint',
+                               kernel_manager.parent.parent.yarn_endpoint)
+        self.yarn_endpoint_security_enabled \
+            = proxy_config.get('yarn_endpoint_security_enabled',
+                               kernel_manager.parent.parent.yarn_endpoint_security_enabled)
         yarn_master = urlparse(self.yarn_endpoint).hostname
-        self.resource_mgr = ResourceManager(address=yarn_master)
+        if self.yarn_endpoint_security_enabled is True:
+            print('>>> SECURITY ENABLED')
+            self.resource_mgr = ResourceManager(address=yarn_master, kerberos_enabled=self.yarn_endpoint_security_enabled)
+        else:
+            print('>>> SECURITY DISABLED')
+            self.resource_mgr = ResourceManager(address=yarn_master)
+        self.resource_mgr.logger.setLevel('DEBUG')
```

Comment:
Could we please remove the print statements - or replace the first with a log statement to the effect of "YARN endpoint security is enabled"?
Also, I believe the resource_mgr DEBUG level was inadvertently left in the code since we go out of our to adjust the base level to WARNING since its way too noisy otherwise.